chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -450,7 +451,8 @@ public synchronized void saveScene(RenderContext context, TaskTracker taskTracke
     try (TaskTracker.Task task = taskTracker.task("Saving scene", 2)) {
       task.update(1);
 
-      try (BufferedOutputStream out = new BufferedOutputStream(context.getSceneDescriptionOutputStream(name))) {
+      getSceneDirectory();
```

Comment:
What does `getSceneDirectory();` do? If it has side effects, it should be renamed so that its name reflects what it does (i.e. not only getting something). Maybe `ensureSceneDirectory()`?